cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -1,33 +1,19 @@
-import json
-import os
-
 import anndata as ad
-
+import pandas as pd
 from . import utils
-
-BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+import dask
 
 # fmt: off
 # ONTOLOGY TERMS TO UPDATE ACROSS ALL DATASETS IN CORPUS
-# Initialization is AUTOMATED for newly deprecated terms that have 'Replaced By' terms in their ontology files
```

Comment:
Let's keep these types of comments so this file can remain as a template for future migrations